Write part on Array helper for 3.8 release blog (#3816)
emberjs/website: source/blog/2019-02-18-ember-3-8-released.md

> Second new feature (2 of 2)

Reviewer: ```suggestion
Array helper (2 of 2)
```

> Array helper

Reviewer: ```suggestion

```

Ember 3.8 introduces the `{{array}}` helper to create an array in a template. This helper works in similar fashion to the already existing `{{hash}}` helper.

The helper would be invoked as `{{array arg1 ... argN}}` and return the value `[arg1, ..., argN]`. For example, `{{array 'a' 'b' 'c'}}` would return the value `['a', 'b', 'c']`.

> You can read the original RFC [here](https://github.com/emberjs/rfcs/blob/master/text/0318-array-helper.md).

Reviewer: I tend to not link on words like "here", "click here" etc. since it can be confusing for people using assistive technologies like screen readers. 

```suggestion
Read the [original RFC](https://github.com/emberjs/rfcs/blob/master/text/0318-array-helper.md) for more information.
```
